sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Utility/ReporterUtils.cs

```diff
@@ -80,7 +81,18 @@ internal static string GetRunName(CIInfo ciInfo)
 
         internal static async Task<string> RunCommandAsync(string command, bool async = false)
         {
-            var processInfo = new ProcessStartInfo("cmd", $"/c {command}")
+            string shell, shellArgs;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
pls check if this required in any other place. BTW, do we have similar logic in js package ?

Reply:
This is the only place we run command in a separate process. The JS package does not require specifying shell type.